### PR TITLE
feat(ui5-tree-item): implement info/infoState attributes

### PR DESCRIPTION
**Pull request by yanaminkova**
Info and infoState attributes are added to ui5-tree-item.

Fixes: #2633

### CODE REVIEW

**Comment by nnaydenow** [comment]
Hi @ilhan007,

Looks good to me. Could you please check and approve?

Regards,
Nayden

File: packages/main/src/TreeItem.js [hunk 1]
```diff
@@ -66,6 +67,30 @@ const metadata = {
 		icon: {
 			type: String,
 		},
+
+		/**
+		 * Defines the <code>info</code>, displayed in the end of the tree item.
+		 * @type {string}
+		 * @public
+		 * @since 1.0.0-rc.11
```
Comment (ilhan007) [inline packages/main/src/TreeItem.js:75]: RC11 is already released, these two will go to RC12

File: packages/main/src/themes/TreeListItem.css [hunk 2]
```diff
@@ -83,3 +83,41 @@
...
+	margin: 0 0.25rem;
+	color: var(--sapNeutralTextColor);
+	font-size: 0.875rem;
+	min-width: 6rem;
+	max-width: 40%;
+	white-space: nowrap;
+	overflow: hidden;
+	text-overflow: ellipsis;
+}
+.ui5-li-wrapper {
```
Comment (ilhan007) [inline packages/main/src/themes/TreeListItem.css:115]: minor thing, but could you add a new line to separate these selectors

File: packages/main/src/themes/TreeListItem.css [hunk 3]
```diff
@@ -83,3 +83,41 @@
 :host([active][actionable]) .ui5-li-tree-toggle-icon {
     color: var(--sapList_Active_TextColor);
 }
+
+/* [ui5-li]: infoState */
+:host([info-state="Warning"]) .ui5-li-info {
```
Comment (ilhan007) [inline packages/main/src/themes/TreeListItem.css:88]: ui5-li-info is not appropriate naming as this the tree item - ui5-li-tree-info is more suitable for a class name.

File: packages/main/src/TreeListItem.js [hunk 4]
```diff
@@ -71,6 +72,30 @@ const metadata = {
 			type: Boolean,
 		},
 
+		/**
+		 * Defines the <code>info</code>, displayed in the end of the tree item.
+		 * @type {string}
+		 * @public
+		 * @since 1.0.0-rc.11
```
Comment (ilhan007) [inline packages/main/src/TreeListItem.js:79]: don't forget the since here as well

**Review by ilhan007 — CHANGES_REQUESTED** [review]
Hello @yanaminkova Yana, thanks for the change. Overall, it is what I expected. Could you just document what is done with this change as part of the commit body. And besides that, there are few comments you can find inline.

File: packages/main/src/TreeListItem.hbs [hunk 5]
```diff
@@ -18,9 +18,15 @@
 {{/inline}}
 
 {{#*inline "listItemContent"}}
-	{{#if _showTitle}}
-		<div part="title" class="ui5-li-title"><slot></slot></div>
-	{{/if}}
+	<div class="ui5-li-wrapper">
```
Comment (ilhan007) [inline packages/main/src/TreeListItem.hbs:21]: Update this class name as well - ui5-tree-item-text-wrapper

**Comment by yanaminkova** [comment]
Hi @ilhan007 , the comments are fixed and the commit body is updated as well.

**Review by ilhan007 — APPROVED** [review]

**Pull request merged**